packages/server/prisma/schema.prisma

```diff
@@ -17,7 +17,7 @@ model USER {
   username    String?
```

Comment:
Not sure how much this changes. Feel free to delete and regen migrations because there not run yet. But it probable should be:
```
model User {
```